deployment/terraform/ad-manager/templates/ad-manager_conf.tpl

```diff
@@ -18,9 +18,9 @@ ad-manager {
 
  # Graphite data source config
   graphite-base-uri = "${graphite_base_uri}"
-  graphite-earliest-time ="${graphite_earliest_time}"
-  graphite-max-data-points = "${graphite_max_data_points}"
+  graphite-bin-size-in-mins = "${graphite_bin_size_in_mins}"
```

Comment:
Can the bin size be derived from the metric function string?
If so, that's definitely preferable.
If not, please make this to be measured in `seconds` instead of `mins`.